### DISCUSSION TITLE
FreeBSD Support

### DISCUSSION THREAD

**Discussion by tschettervictor** [openrport/openrport#59] [Roadmap Feature Wishes Idea]
Are there plans to support FreeBSD with the install script?
I'm able to build openrport perfectly, but having trouble getting things up and running.

The database for example needs the schema to be manually created.

**Comment by alhdo** [comment]
Need some works to make it seamless in FreeBSD but since we are focusing on releasing the new front-end. This is not the focus for the moment. However you are free to open a PR on the repo rport-installer available on the organization repository.

**Comment by tschettervictor** [comment]
Are there any docs explaining the setup process? Or does one have to go through the install script and try to follow along?ThanksVictorOn Nov 20, 2024, at 09:03, Castro Alhdo ***@***.***> wrote:﻿
Need some works to make it seamless in FreeBSD but since we are focusing on releasing the new front-end. This is not the focus for the moment. However you are free to open a PR on the repo rport-installer available on the organization repository.

**Reply by alhdo** [reply]
I am afraid you have to go to the script but there's comment to help you through.
